common/src/main/java/com/ssblur/scriptor/helpers/SpellbookHelper.java

```diff
@@ -36,12 +36,14 @@ public static boolean castFromItem(ItemStack itemStack, Player player) {
         player.sendSystemMessage(Component.translatable("extra.scriptor.fizzle"));
         ScriptorAdvancements.FIZZLE.get().trigger((ServerPlayer) player);
         if(!player.isCreative())
-          addCooldown(player, 350);
+          addCooldown(player, (int) Math.round( 350.0D * ( (double) level.getGameRules().getInt(ScriptorGameRules.TOME_COOLDOWN_MULTIPLIER) / (double) 100) ));
         return true;
       }
       spell.cast(new SpellbookTargetable(itemStack, player, player.getInventory().selected).withTargetItem(false));
-      if(!player.isCreative())
-        addCooldown(player, (int) Math.round(spell.cost() * 7));
+      if(!player.isCreative()) {
+		double adjustedcost = spell.cost() * (double) ( (double) level.getGameRules().getInt(ScriptorGameRules.TOME_COOLDOWN_MULTIPLIER) / (double) 100);
```

Comment:
Change adjustedcost to camelCase (adjustedCost)

Reply:
Alright. Will do.